packages/graph/src/graph2/graph.ts

```diff
@@ -968,6 +987,10 @@ export class Graph2 extends SVGZoomWidget {
         }
     }
 
+    tooltipHTML(data) {
+        return data.props[this.tooltipContentColumn()];
+    }
```

Comment:
Users can either overwrite this method or use `tooltipContentColumn` (or both if they want to rely on tooltipContentColumn within their custom tooltipHTML method).